pyproject.toml

```diff
@@ -29,8 +32,10 @@ tiktoken = {version = "^0", optional = true, python="^3.9"}
 pinecone-client = {version = "^2", optional = true}
 weaviate-client = {version = "^3", optional = true}
 google-api-python-client = {version = "2.70.0", optional = true}
+freezegun = "^1.2.2"
```

Comment:
whats this needed for?

Reply:
unit tests. Since the tracer sets the start time and end time of each run using utcnow, this library overrides that for tests so we can test of equality. probably should add it as a test dep tho

Reply:
yup test dep probably best!